llvm/lib/Target/AMDGPU/Utils/AMDGPUBaseInfo.h

```diff
@@ -1367,6 +1367,28 @@ constexpr bool isEntryFunctionCC(CallingConv::ID CC) {
   }
 }
 
+// Shaders that are entry functions need to count input arguments even if
```

Comment:
I still think we should just do this for all entry points and not special case the graphics shaders

Reply:
When you do reordering and reparenting, graphite is trying to create temporary branch names that don't respect the users/username prefix rule